### PR TITLE
Fix static option display and assert form key error on submit.

### PR DESCRIPTION
**Pull request by jitendrapurohit**
Overview
----------------------------------------
Fix static option display and assert form key error on submit.

Before
----------------------------------------
No options displayed on selecting static option.

After
----------------------------------------
Displayed correctly.

image

Comments
----------------------------------------
`$this->rootUser` is provided by core drupal. This is also used by `webform` module in verifying all the test executions so it should be fine to use it in our test setup too? @KarinG

### CODE REVIEW

File: tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php [hunk 1]
```diff
@@ -255,13 +259,15 @@ public function testSubmitWebform() {
     $this->getSession()->getPage()->checkField('Yes');
 
     $this->getSession()->getPage()->pressButton('Submit');
+    $this->assertPageNoErrorMessages();
```
Comment (jitendrapurohit) [inline tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php:262]: No error after submit verifies there was no form key error after webform was submitted.

File: tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php [hunk 2]
```diff
@@ -219,15 +219,15 @@ public function testSubmitWebform() {
...
-    // $this->htmlOutput();
+    $checkbox_edit_button = $this->assertSession()->elementExists('css', '[data-drupal-selector="edit-webform-ui-elements-civicrm-1-contact-1-cg1-custom-3-operations"] a.webform-ajax-link');
+    $checkbox_edit_button->click();
+    $this->assertSession()->assertWaitOnAjaxRequest();
+    $this->htmlOutput();
+    $this->getSession()->getPage()->selectFieldOption("properties[civicrm_live_options]", 0);
+    $this->assertSession()->assertWaitOnAjaxRequest();
+    $this->assertSession()->waitForField('properties[options][options][civicrm_option_1][label]');
+    $this->getSession()->getPage()->fillField('properties[options][options][civicrm_option_1][label]', 'Red - Recommended');
+    $this->htmlOutput();
```
Comment (jitendrapurohit) [inline tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php:230]: Confirms the static radio button displays the option correctly after ajax refresh.

File: tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php [hunk 3]
```diff
@@ -179,7 +179,7 @@ public function testSubmitWebform() {
 
     $this->createCustomFields();
 
-    $this->drupalLogin($this->adminUser);
+    $this->drupalLogin($this->rootUser);
     $this->drupalGet(Url::fromRoute('entity.webform.civicrm', [
```
Comment (KarinG) [inline tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php:183]: I looked further and I still don't know which permission our `adminUser` was missing (and was therefore losing form_key while Editing an Element with Options) - but `rootUser` is 👍

File: tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php (hunk 2 above)
Comment (KarinG) [reply tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php:230]: After merging this I'll finish this test -> next we want to see Red - Recommended on the webform.

File: tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php [hunk 4]
```diff
@@ -255,13 +259,15 @@ public function testSubmitWebform() {
...
     $this->assertSession()->pageTextContains('New submission added to CiviCRM Webform Test.');
 
     // Note: custom fields are on contact_id=3 (1=default org; 2=the drupal user)
     $utils = \Drupal::service('webform_civicrm.utils');
+    $contactID = $utils->wf_civicrm_api('Contact', 'get', $params)['id'];
     $api_result = $utils->wf_civicrm_api('CustomValue', 'get', [
       'sequential' => 1,
-      'entity_id' => 3,
+      'entity_id' => $contactID,
     ]);
```
Comment (KarinG) [inline tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php:271]: 👍

**Pull request merged**